src/auxinfo/mod.rs

```diff
@@ -10,5 +10,5 @@
 
 pub(crate) mod auxinfo_commit;
 pub(crate) mod info;
-pub(crate) mod participant;
+pub mod participant;
```

Comment:
The `status` associated type of implementors of `ProtocolParticipant` is not currently very useful. It is not exposed to users, so there is not much that I can do with them. I exposed them for this example, but ended up not actually using it.

Reply:
This is solved by this PR.